plasmapy/particles/particle_class.py

```diff
@@ -273,51 +273,16 @@ class Particle:
 
     """
 
-    def __init__(
-            self,
+  
+   ##############
+
+    def __checkingAlias__(self,
```

Comment:
[PEP8: for methods we use lowercase with underscore spacing](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)

This method also does not reach the standard for a dunder, so a single leading underscore will suffice, i.e. ``__checkingAlias__ -> _checking_alias``